db/projects/neo/project.yaml

```diff
@@ -0,0 +1,168 @@
+name: NEO
+web:
+  - 'https://neo.org/'
+start_date: '2014-02-01'
+description: 'NEO is a blockchain, originally known as AntShares, sometimes called as chinese Ethereum. It is open source platform for smart contracts in China.'
```

Comment:
"platform for smart contracts in China" - that's only prejudice. Is there any evidence that its platform specifically for China?
Origin of developers its not relevant in open-source & decentralized ledgers.

Reply:
OK, it's my typo. The original idea was:
It is first open source platform for smart contracts in started in China.

Reply:
Sources:
https://www.financemagnates.com/cryptocurrency/bloggers/whats-big-deal-chinas-first-open-source-blockchain-platform-neo/
https://medium.com/on-the-origin-of-smart-contract-platforms/on-the-origin-of-neo-7f9817849af3
https://www.coinspeaker.com/2017/07/24/chinas-first-open-source-blockchain-platform-antshares-rebrands-to-neo/